app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+  before_filter :require_user
+	def create
+    #binding.pry
+    @video = Video.find(params[:video_id])
+    review = Review.new(review_params)#@review = Review.new(review_params)
+    #review = @video.reviews.build(review_params.merge!(user: current_user))
```

Comment:
Why split this up? While the one line style requires a understanding of slightly more advanced methods `build` & `merge!` it is still very readable in one line. When you make decisions sure you are prepared to defend them because people will challenge them.